### PR TITLE
*: RC1 update

### PR DESCRIPTION
**Pull request by AnnaShaleva**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module github.com/nspcc-dev/neo-go-sc-wrkshp/v2
```
Comment (roman-khimov) [inline go.mod]: v2?

File: 2-storage.yml [hunk 2]
```diff
@@ -5,4 +5,3 @@ events:
     parameters:
       - name: message
         type: ByteString
-
```
Comment (roman-khimov) [inline 2-storage.yml:8]: ?

File: go.mod (hunk 1 above)
Comment (AnnaShaleva) [reply go.mod]: Fixed.

File: 2-storage.yml (hunk 2 above)
Comment (AnnaShaleva) [reply 2-storage.yml:8]: This yaml had two empty lines at the end, I removed one of them.

**Comment by AnnaShaleva** [comment]
Do we need to remove the wallet from the workshop after https://github.com/nspcc-dev/neo-go/pull/1854?

**Comment by roman-khimov** [comment]
Let it stay here for now.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**